_includes/js/custom.js

```diff
@@ -99,4 +69,383 @@ jQuery(document).ready(function() {
+                foundry_order = `
+                        <a class="col-sm-1 btn btn-default btn-sm" href="/principles/fp-000-summary.html" aria-label="View the OBO Foundry criteria for the Foundry status of ${title}" title="View the OBO Foundry criteria for the Foundry status of ${title}">
+                               <span class="glyphicon glyphicon-star" aria-hidden="true"></span>
+                        </a>             
+                `;
+            }
+            if (activity_status === "inactive") {
+                is_inactive = "inactive_row";
+            }
+            if (data[i]["is_obsolete"]) {
+                is_obsolete = "(obsolete)"
+            }
+            // table row template
+            let template = `
+                <tr class="row ${is_inactive}">
+                    <td class="col-sm-1">
+                        <a class="" href="ontology/${id}.html">
+                           ${id} 
+                        </a>
+                        <span style="background-color: #ff8d82">${is_obsolete}</span>    
+                    </td>
+                    <td class="col-sm-1">
+                        ${title}
+                    </td>
+                    <td class="col-sm-3">
+                        ${description}
+                        <small>
+                            <a href="ontology/${id}.html">Detail</a>
+                        </small>
+                    </td>
+                    <td class="col-sm-5">
+                        <div class="ic-display">
+                            <a class="btn btn-default btn-sm" href="ontology/${id}.html" aria-label="More details for ${title}" title="More details for ${title}">
+                                  <span class="glyphicon glyphicon-info-sign" aria-hidden="true"></span>
+                            </a>
+                            <a class="btn btn-default btn-sm" href="${homepage}" aria-label="Go to the homepage for ${title}" title="Go to the homepage for ${title}">
+                                 <span class="glyphicon glyphicon-home" aria-hidden="true"></span>
+                            </a>
+                            <a class="btn btn-default btn-sm" href="http://purl.obolibrary.org/obo/${id}.owl" aria-label="Download ${title} in OWL format" title="Download ${title} in OWL format">
+                                <span class="glyphicon glyphicon-download" aria-hidden="true"></span>
+                            </a>
+                            <a class="btn btn-default btn-sm" href="http://www.ontobee.org/browser/index.php?o=${id}" aria-label="Browse ${title} on OntoBee" title="Browse ${title} on OntoBee">
+                                <span class="glyphicon glyphicon-eye-open" aria-hidden="true"></span>
+                            </a>
+                            ${tracker}
+                            ${contact}
+                            ${publication}
+                            ${foundry_order}
```

Comment:
there are two issues with this star:
1. It potentially makes the icons not line up horizontally, as seen on the top of the page with BFO
<img width="1110" alt="Screen Shot 2022-07-31 at 14 52 06" src="https://user-images.githubusercontent.com/5069736/182027319-c7e73fac-7d4b-4b4f-8588-822b2f9d629e.png">
2. None of these icons are super informative (not an issue with this PR- it was already that way)


Can we change it to instead add an emoji to the right of the prefix in the ID column like ❤️ or ✨ that is less obtrusive

Reply:
Oops sorry I did not even see this. Very important, the star button should be removed entirely. It is no longer needed at all.